@types/jsb.d.ts

```diff
@@ -1411,9 +1411,9 @@ declare namespace jsb {
         isExternalOfferAvailableAsync: (listener: ExternalOfferAvailabilityListener) => void;
 
         isFeatureSupported: (productType: string) => BillingResult;
-        showAlternativeBillingOnlyInformationDialog: (listener: AlternativeBillingOnlyInformationDialogListener) => BillingResult;
```

Comment:
These interfaces are required to be executed on the UI thread, so it is not possible to get the return value